docs/src/pages/components/material-icons/SearchIcons.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Friendly nit: Please annotate each line. Otherwise this could hide legitimate issues.